stubs/pysftp/pysftp/__init__.pyi

```diff
@@ -0,0 +1,125 @@
+from stat import S_IMODE as S_IMODE
+from types import TracebackType
+from typing import IO, Any, Callable, List, Optional, Sequence, Text, Tuple, Type, Union
+from typing_extensions import Literal
+
+import paramiko
+from paramiko import AuthenticationException as AuthenticationException
+from pysftp.exceptions import (
+    ConnectionException as ConnectionException,
+    CredentialException as CredentialException,
+    HostKeysException as HostKeysException,
+)
+from pysftp.helpers import (
+    WTCallbacks as WTCallbacks,
+    cd as cd,
+    known_hosts as known_hosts,
+    path_advance as path_advance,
+    path_retreat as path_retreat,
+    reparent as reparent,
+    st_mode_to_int as st_mode_to_int,
+    walktree as walktree,
+)
+
+class CnOpts:
+    log: bool = ...
+    compression: bool = ...
+    ciphers: Sequence[str] = ...
+    hostkeys: paramiko.HostKeys = ...
+    def __init__(self, knownhosts: Optional[str] = ...) -> None: ...
+    def get_hostkey(self, host: str) -> paramiko.PKey: ...
+
+_Callback = Callable[[int, int], Any]
+_Path = Union[Text, bytes]
```

Comment:
Should this include other PathLike objects?

Reply:
Well, the underlying paramiko doesn't receive PathLike and since this is a wrapper, I used the same.
If paramiko ever changes its signatures, it will be useful to do the same here.

Ideally, we could just import a type from there.